features/inventory/managing_inventory/filtering_inventory_by_code.feature

```diff
@@ -0,0 +1,24 @@
+@managing_inventory
+Feature: Filtering inventory by a code
```

Comment:
I guess we don't need an article here as it's not a concrete, single code - "by ~a~ code"